src/lesson_02/hometask_02/TaskSix.java

```diff
@@ -0,0 +1,47 @@
+package lesson_02.hometask_02;
+
+import java.util.Scanner;
+
+public class TaskSix {
+    public static void main(String[] args) {
+        // Using Scanner
+        Scanner sc = new Scanner(System.in);
+        ////// Task 6
+        // User inputs 4-digit natural number [1000 - 9999]
+        System.out.println("Task 6:");
+        System.out.println("Please enter 4-digit natural number [1000 - 9999]: ");
+        int number = sc.nextInt();
+
+        // Separate 4th digit:
+        int digit4 = number % 10;
```

Comment:
There is more simple way:
        System.out.println(number % 10);
        System.out.println((number % 100)/10);
        System.out.println( (number % 1000) / 100);
        System.out.println(number / 1000);